### PR TITLE
page for careers

### PR DESCRIPTION
**Pull request by hendrikebbers**

### CODE REVIEW

**Comment by hendrikebbers** [comment]
@madmas @timo0 @giftkugel @sclassen some input / ideas for the text would be awesome :)

**Comment by hendrikebbers** [comment]
I actually like "You@Karakun" as headline / topic and hope itÄs much cooler than "careers" ;) We can use you@karakun.com as mail adreess for that topic :)

File: become-a-karakuni.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+The following image shows just some of the technologies and open source projects that we at Karakun use in
+our daily business. Since we love open source we already contributed to several of that technologies.
+
+![technologies]({{ "/assets/images/pages/become-a-karakuni/technologies.png" | absolute_url }})
+
+TODO: Schulung, Vorträge, Mit Autoren & JCs zusammen arbeiten. Contribution to open source
+
+## Find the right location
+
+TODO: Basel oder Dortmund
```
Comment (timo0) [inline become-a-karakuni.md]: Or Stuttgart :P

Comment (hendrikebbers) [reply become-a-karakuni.md]: No :P Currently only for the 2 headquarters

File: become-a-karakuni.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: article
+title: You@Karakun
+order: '19'
+headerText: 
+permalink: /become-a-karakuni/
```
Comment (madmas) [inline become-a-karakuni.md]: Can we please either stick with Karakunie or change it to "Karakun". I think the -ie belittlement is not helpful for our representation.

File: become-a-karakuni.md (hunk 1 above)
Comment (madmas) [reply become-a-karakuni.md]: We only have one headquater - Basel ;-)

File: become-a-karakuni.md (hunk 2 above)
Comment (hendrikebbers) [reply become-a-karakuni.md]: makes sense ;)

**Comment by hendrikebbers** [comment]
@madmas @timo0 I'm done with the text. Can you please have a look and send feedback :)

**Comment by hendrikebbers** [comment]
I ordered an image at the designer for the page in addition :)

**Comment by hendrikebbers** [comment]
@sclassen got it :) I created that image some time ago for a customer presentation and just added it. Will remove the image. I think for a first version we are fine than ;)

**Comment by hendrikebbers** [comment]
The GL wants a more clear message about Karakun. We have a meeting about that topic next week.

**Pull request merged**